apps/frontend/src/libs/maps/color-to-css.map.ts

```diff
@@ -0,0 +1,19 @@
+import { type Color } from "~/libs/types/types.js";
```

Comment:
Instead of this, let's go with another approach. Let's add styles file to Icon component and set it's fill or color to currentColor. Then add need color to Icon wrapper

Reply:
If I correctly understand, `IconButton` is an expample of such Icon wrapper, and I should set icon color inside `IconButton` styles file instead of passing it as a prop to `Icon` component. Right?

But then the question is how to control the color of `IconButton`'s icon? Sometimes it should have `text-primary` and sometimes `text-secondary`. 

I check figma design again, and come up with conclusion: all IconButtons without border have `text-secondary` color, and only with border have `text-primary`. So does it make sense to stick to this observation and for `primary` button variant use `text-secondary` color, and for `outlined` use `text-primary`? or there must be better solution?

Reply:
> So does it make sense to stick to this observation and for primary button variant use text-secondary color, and for outlined use text-primary? or there must be better solution?

yes, let's stick to it. In case we need more colors we will add a color property, but it only will accept a predefined limited set of colors (not css variables) and then we will match them to appropriate styles. Just as example of what I mean:

```typescript
type Properties = {
  color: 'primary' | 'grey',
}

const classNames = getValidClassNames(
  color === "primary" && styles["color-primary"],
  color === "grey" && styles["color-gray"],
)
```

But for now implement it in simpler way as you described above (without color property)